nav2_behavior_tree/plugins/condition/goal_updated_condition.cpp

```diff
@@ -24,13 +24,16 @@ GoalUpdatedCondition::GoalUpdatedCondition(
   const BT::NodeConfiguration & conf)
 : BT::ConditionNode(condition_name, conf)
 {
+  node_ = config().blackboard->get<rclcpp::Node::SharedPtr>("node");
 }
 
 BT::NodeStatus GoalUpdatedCondition::tick()
 {
+  RCLCPP_INFO(node_->get_logger(), "Goal updated: Ticked");
```

Comment:
Probably should remove the logging now :smile:

Reply:
haha. Done